client/views/windows/connect.tpl

```diff
@@ -69,6 +63,12 @@
 			<input class="input username" id="connect:username" name="username" value="{{defaults.username}}">
 		</div>
 		{{/unless}}
+		<div class="col-sm-3">
+			<label for="connect:password">Password</label>
+		</div>
+		<div class="col-sm-9">
+			<input class="input" id="connect:password" type="password" name="password" value="{{defaults.password}}">
```

Comment:
I believe this default needs to be added to server.js's getClientConfiguration() no?

(I don't know if having a default password makes sense but that's a different story)

Reply:
You are right, added.